### PR TITLE
Typography: Text-styling

### PR DESCRIPTION
**Pull request by Heavenlydawn**
Done with the styling of the typography

### CODE REVIEW

File: src/app/[locale]/globals.css [hunk 1]
```diff
@@ -7,7 +7,89 @@
...
+	}
+	.vm-text-p4-medium{
+		@apply text-xs font-medium leading-7
+	}
+	.vm-text-p5-regular{
+		@apply text-[10px] font-normal leading-7
+	}
+	.vm-text-p5-medium{
+		@apply text-[10px] font-medium leading-7
+	}
```
Comment (zorro-ftw) [inline src/app/[locale]/globals.css:94]: Do we have these in the design? If not, please remove.

**Review by zorro-ftw — CHANGES_REQUESTED** [review]
Please follow commit-lint rules for commit messages and PR titles.

example commit message & PR title:
`feat: typography`

**Comment by zorro-ftw** [comment]
For future references, please try to match branch names as closely as possible to commit messages:
A good branch name example would be:

`feat/typography`

**Comment by Heavenlydawn** [comment]
> Please follow commit-lint rules for commit messages and PR titles.
> 
> example commit message & PR title: `feat: typography`

Noted I will do that now

File: src/app/[locale]/globals.css (hunk 1 above)
Comment (Heavenlydawn) [reply src/app/[locale]/globals.css:94]: We do.
it is in the paragraph section of the typography component

Comment (zorro-ftw) [reply src/app/[locale]/globals.css:94]: Oh I see, the file had two P1 stylings. Ok we can resolve this one

Comment (Heavenlydawn) [reply src/app/[locale]/globals.css:94]: Yeah it does.
One for regular font-weight, the other for medium font-weight

**Pull request closed without merging**